src/Generator/DocBlock/Tag/PropertyTag.php

```diff
@@ -37,7 +37,7 @@ public function getName()
      */
     public function setPropertyName($propertyName)
     {
-        $this->propertyName = ltrim($propertyName, '$');
+        $this->propertyName = ! empty($propertyName) ? ltrim($propertyName, '$') : '';
```

Comment:
```suggestion
        $this->propertyName = ltrim((string) $propertyName, '$');
```

Reply:
As you can see other methods also have this expression.

Reply:
https://github.com/laminas/laminas-code/pull/207#issuecomment-3429629936

Reply:
That's nothing todo with strict_types. That's php feature of deprecating `null` support on string functions on php 8.1+.

nullable property is related with initialization, while filling its value should follow the doc as a signature.

The empty value already checked on construct via empty, when filling via setter, exact string per signature should be followed.